build_rules.gradle

```diff
@@ -518,24 +518,33 @@ ext.applyAvroNature = {
 
 // A class defining the set of configurable properties for createJavaQuickstartValidationTask
 class JavaQuickstartConfiguration {
-  // Name for the quickstart is required.
-  // Used both for the test name runQuickstartJava${name}
-  // and also for the script name, quickstart-java-${name}.toLowerCase().
-  String name
+  // Type [Quickstart, MobileGaming] for the postrelease validation is required.
+  // Used both for the test name run${type}Java${runner}
+  // and also for the script name, ${type}-java-${runner}.toLowerCase().
+  String type
 
-  // gcpProject sets the gcpProject argument when executing the quickstart.
+  // runner [Direct, Dataflow, Spark, Flink, FlinkLocal, Apex]
+  String runner
+
+  // gcpProject sets the gcpProject argument when executing examples.
```

Comment:
Is this required for all of the examples?

Certainly the new ones look as though they're only appropriate for the MobileGaming examples. Can we fork at least some parts of the two configs?

Reply:
They are used by both Quickstart and MobileGaming except for "dataset", which is MobileGaming specific.